woocommerce-payments.php

```diff
@@ -12,7 +12,7 @@
  * WC tested up to: 7.7.0
  * Requires at least: 6.0
  * Requires PHP: 7.3
- * Version: 5.8.1
+ * Version: 5.9.0
```

Comment:
We probably don't want to bump the version number as part of this PR.

Reply:
Fixed on [a6ad876](https://github.com/Automattic/woocommerce-payments/pull/6221/commits/a6ad876d640c78682006402f87f7d4412d3674d2). Thanks for catching this!